darwin/utils.py

```diff
@@ -57,7 +57,8 @@ def prompt(msg: str, default: Optional[str] = None) -> str:
 
 
 def find_files(
-        root: Path,
+        root: Optional[Path] = None,
```

Comment:
When is this function called with `root=None`?

Reply:
The cli sends a list of files instead of a root